documentation/decisions/adr/000n-ticket-tracking.md

```diff
@@ -0,0 +1,199 @@
+
+## Context and Problem Statement
+
+The project needs a system for tracking ongoing development work within the project, preferably as a series of tickets that can be organized into sprints. This system would both enable internal stakeholders to prioritize key tasks and assignments throughout the project and help communicate those priorities to external stakeholders.
+
+The goal of this ADR is to evaluate a series of ticket tracking systems and select the one we will be using for the project.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Ticket Tracking:** Tickets can be organized into sprints and tracked as part of larger milestones or epics
+- **Public Access:** Without logging in, members of the public can see tickets that are being worked on
+- **Public Requests:** Members of the public can submit bug reports and feature requests and track how that work is being prioritized
+- **Templates:** The system supports default templates for different types of tickets which prompts the person creating the ticket for a specific set of information
+- **Authority to Operate (ATO):** The platform already must be authorized under the Grants.gov ATO (Authority to Operate) or ATO coverage must be requested
+
+#### Nice to Have
+
+- **Level of Effort Estimates:** Tickets can be assigned an estimated level of effort (e.g. story points, t-shirt size, etc.)
+- **Reporting:** Users can report on key metrics like burndown, point allocation, etc. from directly within the tool
+- **Custom Views:** Users can create custom views for managing tickets with multiple layouts (e.g. kanban board, tabular, roadmap)
+- **Custom Fields:** Users can create custom fields and views to manage their projects
+- **Automation:** Users can automate basic workflows like adding and moving tickets, linking PRs to their originating PRs, etc.
+- **Open Source:** The tool used to manage and host the wiki content should be open source, if possible
+
+## Options Considered
+
+- [GitHub Issues + Zenhub](#github-issues--zenhub)
+- [GitHub Issues + GitHub Projects](#github-issues--github-projects)
+- [Jira](#jira)
+- [OpenProject](#openproject)
+
+## Decision Outcome <!-- REQUIRED -->
+
+We are planning to use **GitHub issues with GitHub projects** because it is the only solution that allows members of the public to submit feature requests and bug reports and then track how those issues are being prioritized within upcoming sprints. Additionally, the use of these tools is free with public repositories and offers other helpful features such as custom fields and views.
+
+**NOTE:** Given some of the constraints around reporting, we may want to continue to explore options that extend the reporting capabilities of GitHub projects. Similarly, if team productivity is meaningfully affected by the absence of the more robust features that Jira or Zenhub offers, we may want to re-evaluate this decision after an initial trial period.
```

Comment:
Yeah I think making a plan for reporting (as @EOKENAVA noted) would be good. 

Burndown charts for the sprint and burndown charts for each milestone are probably the biggest reporting needs. But also as @EOKENAVA said, points-per-developer during sprint planning is a very very useful view, as well as average sprint velocity for the entire team, etc. Maybe we can document all of the must-have and nice-to-have reporting requirements? 

One option would even be to script a reporting tool -- like I could imagine something that's not *too* hard to write that pulls in all GitHub milestones, checks what % of the tasks underneath those milestones is complete, generates a PDF with a burndown chart for each, and posts that burndown chart to Slack or something every day. 

There are also some public tools like https://github.com/thehale/github-projects-burndown-chart or https://github.com/alessioarcara/burndown_chart. 

But maybe there are some other options that already do this outside of manually creating tools. Or maybe it's just worth having a few ZenHub licenses for the internal team, and the public can view tickets in GitHub.